packages/playground/src/components/Playground.tsx

```diff
@@ -0,0 +1,366 @@
+    setSubtheme(subtheme);
+    setStylesheet(stylesheet);
+  };
+
+  const onValidatorSelected = (validator: string) => {
+    setValidator(validator);
+  };
+
+  const handleSetLiveSettings = ({ formData }: { formData: any }) => {
+    setLiveSettings(formData);
+  };
+
+  const onFormDataChange = ({ formData = '' }, id: string) => {
+    if (id) {
+      console.log('Field changed, id: ', id);
+    }
+
+    setFormData(formData as any);
+    setShareURL(null);
+  };
+
+  const onShare = () => {
+    const {
+      location: { origin, pathname },
+    } = document;
+    try {
+      const hash = btoa(
+        JSON.stringify({
+          formData,
+          schema,
+          uiSchema,
+          theme,
+          liveSettings,
+        })
+      );
+
+      setShareURL(`${origin}${pathname}#${hash}`);
+    } catch (err) {
+      setShareURL(null);
+    }
+  };
+
+  let templateProps: any = {};
+  if (ArrayFieldTemplate) {
+    templateProps.ArrayFieldTemplate = ArrayFieldTemplate;
+  }
+  if (ObjectFieldTemplate) {
+    templateProps.ObjectFieldTemplate = ObjectFieldTemplate;
+  }
+  if (extraErrors) {
+    templateProps.extraErrors = extraErrors;
+  }
+
+  return (
+    <>
+      <div className='page-header'>
+        <h1>react-jsonschema-form</h1>
+        <div className='row'>
+          <div className='col-sm-6'>
+            <Selector onSelected={load} />
+          </div>
+          <div className='col-sm-2'>
+            <Form
+              idPrefix='rjsf_options'
+              schema={liveSettingsSchema as any}
+              formData={liveSettings}
+              validator={localValidator}
+              onChange={handleSetLiveSettings as any}
+            >
+              <div />
+            </Form>
+          </div>
+          <div className='col-sm-2'>
+            <ThemeSelector themes={themes} theme={theme} select={onThemeSelected} />
+            {themes[theme] && themes[theme].subthemes && (
+              <SubthemeSelector subthemes={themes[theme].subthemes} subtheme={subtheme} select={onSubthemeSelected} />
+            )}
+            <ValidatorSelector validators={validators} validator={validator} select={onValidatorSelected} />
+            <button
+              title='Click me to submit the form programmatically.'
+              className='btn btn-default'
+              type='button'
+              onClick={() => playGroundFormRef.current.submit()}
+            >
+              Prog. Submit
+            </button>
+            <span> </span>
+            <button
+              title='Click me to validate the form programmatically.'
+              className='btn btn-default'
+              type='button'
+              onClick={() => {
+                const valid = playGroundFormRef.current.validateForm();
+                alert(valid ? 'Form is valid' : 'Form has errors');
+              }}
+            >
+              Prog. Validate
+            </button>
+            <div style={{ marginTop: '5px' }} />
```

Comment:
This was the code added in the other PR

```suggestion
              <span> </span>
              <button
                title='Click me to reset the form programmatically.'
                className='btn btn-default'
                type='button'
                onClick={() => this.playGroundForm.current.reset()}
              >
                Prog. Reset
              </button>
              <div style={{ marginTop: '5px' }} />
```